resources/zib/ext-FunctionalMentalStatus.MedicalDevice.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="ext-FunctionalMentalStatus.MedicalDevice" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/ext-FunctionalMentalStatus.MedicalDevice" />
+  <name value="ExtFunctionalMentalStatusMedicalDevice" />
+  <title value="ext FunctionalMentalStatus.MedicalDevice" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="This extension is a reference element to MedicalDevice with ID NL-CM:4.26.5. This is where the MedicalDevice is kept to reduce the disability or disorder." />
```

Comment:
'is kept' = 'wordt bewaard'. Dus ik denk dat je 'is referenced' bedoelt.

Als ik kijk naar wat er in de integration branch als extension descriptions is ingevuld, zou ik misschien zoiets neerzetten:
> An extension to add a reference to zib-MedicalDevice to `Goal.target` in zib-FunctionalOrMentalStatus that is modelled inline in zib-TreatmentObjective.

Reply:
Gefixt